src/modules/link-selector.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Link selector
+ * @author Deux Huit Huit
+ */
+(function ($) {
+
+	'use strict';
+
+	var scope = $('body');
+	var fakeAnchor = $('<a />');
+
+	var update = function () {
+		var currentPath = window.location.pathname;
+		scope.find('[href]').each(function () {
+			var t = $(this);
+			var pathname = '';
+			var matches = [];
+
+			fakeAnchor.prop('href', t.attr('href'));
+			pathname = fakeAnchor.prop('pathname');
+			matches = _.intersection(pathname.split('/'), currentPath.split('/'));
```

Comment:
Tested this and currently current-link-partial is added to every links since it matches "" and "fr" or "en".

Reply:
I added a condition for multilingual websites to ignore the language portion of the URL.